Batch description optional

avdgaag opened the pull request:
At the moment, a batch requires a description. This will output as a "vaste omschrijving record 0020". What I can tell from docs on the Clieop format, this record is optional. The gem requires it.

At the moment I cannot create a clieop file without this record, although I really would like to do so. Would you agree this is a bug? Would you accept a pull request?

moiristo commented:
Agreed, the docs say there should be 0-4 0020 records. Maybe we should allow the description to be an array (max size 4) as well..

avdgaag commented:
Currently the description is just split on newlines, I think? That works good enough for me, although I wouldn't mind an array. A check on the number of lines would be good, though.

Can you remove the description requirement and release a new version of the gem? It would be much appreciated. Let me know if I can help.

wvanbergen commented:
Yeah, this change does seem to make sense. Pull requests are accepted :)

moiristo commented:
We're talking about Clieop::Payment::Batch#l40 right? It should become something as in lines 76 to 80.

Commit:
Make Batch description optional

According to the official specs, the 0020 record is optional. The
gem currently requires it, so users are unable to create clieop files
without this record.

This makes sure the record is no longer required, and that the record is
omitted when not present.

avdgaag commented:
Does this comment trigger Travis to test this pull-request?
